### PR TITLE
[zh] Sync #15282 CNI doc update into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #15282 CNI doc update into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 1]
```diff
@@ -73,213 +102,181 @@ $ istioctl install -f istio-cni.yaml -y
...
-1. 等待所有的 Init 容器成功执行完毕。
-1. Istio Sidecar 跟随 Pod 的其它容器一起启动。
+1. 默认接口 CNI 插件设置 Pod 网络接口并分配 Pod IP。
+1. Istio CNI 插件设置到 Pod 内的 Istio Sidecar 代理的流量重定向。
+1. 所有初始化容器均成功执行并完成。
+1. Istio Sidecar 代理与 Pod 的其他容器一起在 Pod 中启动。
 
-Init 容器在 Sidecar 启动之前执行，这会导致在它们执行期间会有流量丢失。
-可以用以下的一种或所有设置来防止流量丢失：
+Init containers execute before the sidecar proxy starts, which can result in traffic loss during their execution. Avoid this traffic loss with one of the following settings:
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/cni/index.md:246]: This en can be removed.

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 2]
```diff
@@ -73,213 +102,181 @@ $ istioctl install -f istio-cni.yaml -y
...
-为了将应用 Pod 的网络命名空间中的流量重定向至 Istio Sidecar，Istio CNI 插件配置了命名空间的 iptables。
-您可以使用与正常情况相同的 Pod 注解来调整流量重定向参数，例如要包含或排除在重定向之外的端口和 IP 范围。
-有关可用参数，请参阅[资源注解](/zh/docs/reference/config/annotations)。
+为了将应用程序 Pod 网络命名空间中的流量重定向到 Istio 代理 Sidecar 或从 Istio 代理 Sidecar 重定向，
+Istio CNI 插件会配置命名空间的 iptables。您可以使用与平常相同的 Pod 注解来调整流量重定向参数，
+例如要包含或排除在重定向中的端口和 IP 范围。有关可用参数，
+请参阅[资源注解](/zh/docs/reference/config/annotations)。
 
-### 与应用的 Init 容器的兼容性 {#compatibility-with-application-init-containers}
+### 与应用程序初始化容器的兼容性 {#compatibility-with-application-init-containers}
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/cni/index.md:235]:
```suggestion
### 与应用的 Init 容器的兼容性 {#compatibility-with-application-init-containers}
```
Init 容器，这种术语可以全文留用

File: content/zh/docs/setup/additional-setup/cni/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/cni/index.md:246]: Done

File: content/zh/docs/setup/additional-setup/cni/index.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/cni/index.md:235]: All done.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**